File: MyStuff/WebContent/js/Author.js
define(['./Country'], function (Country) {
    'use strict';
    function Author(data) {
    	if (data) {
    		this.id = data.id;
            this.url = data.url;
            this.firstName = data.firstName;
            this.lastName = data.lastName;
            this.country = new Country(data.country);
        }
    };
    return Author;
});

File: MyStuff/src/com/balancedbytes/mystuff/games/Author.java
package com.balancedbytes.mystuff.games;

import javax.xml.bind.annotation.XmlRootElement;

import com.balancedbytes.mystuff.games.rest.RestData;

@XmlRootElement
public class Author extends RestData {
	
	private String firstName;
	private String lastName;
	private Country country;
	
	public String getFirstName() {
		return firstName;
	}
	
	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}
	
	public String getLastName() {
		return lastName;
	}
	
	public void setLastName(String lastName) {
		this.lastName = lastName;
	}
	
	public Country getCountry() {
		return country;
	}
	
	public void setCountry(Country country) {
		this.country = country;
	}
	
}
